### PR TITLE
use imagemagick to set colorspace to sRGB for common image types

### PR DESCRIPTION
**Pull request by jdelbick**
## Description

supporting PR in postprocessing to this: https://git.corp.adobe.com/nui/worker-flite/pull/66
(when we switch worker-flite to [use post-processing](https://git.corp.adobe.com/nui/worker-flite/pull/65), we don't want to lose this bug fix)

In order to cause the least amount of side effects, I only set the colorspace for jpg, jpeg, and png renditions

Fixes # https://jira.corp.adobe.com/browse/NUI-826
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: lib/postprocessing/image.js [hunk 1]
```diff
@@ -227,6 +227,11 @@ async function applyOutputProperties(img, instructions) {
         img.define(`jpeg:extent=${instructions.jpegSize}`);
     }
 
+    // make sure colorspace is sRGB: bug with CMYK files producing CMYK images
+    if (instructions.fmt === "jpg" || instructions.fmt === "jpeg" || instructions.fmt === "png") {
+        img.colorspace('sRGB');
```
Comment (alexkli) [inline lib/postprocessing/image.js:232]: As discussed, we should always set it regardless of output format (unless imagemagick doesn't like it for other file types?).

Comment (bobvanmanen) [reply lib/postprocessing/image.js:232]: What does the function actually guarantee? JPEG, PNG, GIF output compatible with the OOTB workers? If so, it should also define that the result is always RGB.

Comment (alexkli) [reply lib/postprocessing/image.js:232]: Comment explanation would be something like: "Most asset compute renditions are used in digital display context (web, screens), where RGB is required"

Comment (jdelbick) [reply lib/postprocessing/image.js]: I changed it to set `sRGB` for all file formats. I also added a cmykJpg -> gif test to verify this works for gif formats.

**Comment by jdelbick** [comment]
@alexkli yes, we can use the `convert "rendition.png[1x1+0+0]" json:` trick if we don't already examine that in the integration tests

File: lib/postprocessing/image.js (hunk 1 above)
Comment (jdelbick) [reply lib/postprocessing/image.js]: btw, any preference between `sRGB` and `RGB`?

**Comment by alexkli** [comment]
@jdelbick We already do that: `getImageMetadata()` [returns the full metadata json](https://git.corp.adobe.com/nui/integration-tests/blob/master/test/integration.test.js#L990-L995), so colorspace could be easily verified.

**Pull request merged**